.github/CODEOWNERS

```diff
@@ -1,2 +1,3 @@
 * @hors @tplavcic @nmarukovich @ptankov @jvpasinatto @eleo007
 /charts/pmm/ @percona/pmm-review-be  @spron-in
+/charts/everest/ @percona/everest-backend
```

Comment:
@recharte it says there's an error here, possibly related to permissions. Do you know what's wrong? I was under the impression all of us under this team had write access to this repo 🤔